src/app/helpers/tenantHelper.js

```diff
@@ -87,11 +87,16 @@ module.exports = {
             },
             appLogo: function (callback) {
               module.exports.getImagePath(baseUrl, tenantId, 'appLogo.png', callback)
-            }
+            },
+            secondaryLogo: function (callback) {
+              module.exports.getImagePath(baseUrl, tenantId, 'secondaryLogo.png', callback)
+            },
           }, function (err, results) {
             if (err) { }
             responseObj.logo = results.logo
               ? results.logo : baseUrl + '/assets/images/sunbird_logo.png'
+              responseObj.secondaryLogo = results.secondaryLogo ? 
+              results.secondaryLogo : envHelper.SECONDARY_LOGO ? baseUrl + '/assets/images/sunbird_secondarylogo.png': baseUrl + '/assets/images/sunbird_logo.png'
```

Comment:
@sohailamjad12 
- enclose fallback condition in separate block
- ```responseObj.secondaryLogo = results.secondaryLogo ? results.secondaryLogo : (envHelper.SECONDARY_LOGO ? baseUrl + '/assets/images/sunbird_secondarylogo.png': baseUrl + '/assets/images/sunbird_logo.png')```

CC: @mangeshmane @rajeevsathish